src/test/ui/macros/unreachable-format-args.rs

```diff
@@ -0,0 +1,14 @@
+// ignore-emscripten no processes
```

Comment:
Why are you ignoring emscripten in these tests?

Reply:
I did the same as this test [unreachable-fmt-msg.rs](https://github.com/rust-lang/rust/blob/8196e37ac5c1ed611470faa1f73b4af2f7bc7037/src/test/ui/macros/unreachable-fmt-msg.rs#L3)